src/Router/Boot.tsx

```diff
@@ -11,11 +11,11 @@ import { BootProps } from "./types"
 
 export const Boot: React.SFC<BootProps> = ({ children, ...props }) => {
   const globalState = new GlobalState(props)
-
   return (
     <StateProvider inject={[globalState]}>
       <ContextProvider
         relayEnvironment={globalState.state.system.relayEnvironment}
```

Comment:
@mzikherman - i think you need to rebase this from master aftert his merges: https://github.com/artsy/reaction/pull/932